docs/src/content/ignition/docs/guides/_dirinfo.yaml

```diff
@@ -10,3 +10,4 @@ order:
   - /verify
   - /viem
   - /create2
+  - /scripts
```

Comment:
Lets move this one under the advanced section. Or maybe a rename that excludes the word advanced?